tests/test_docker_compose_state_persistence.py

```
@@ -168,57 +169,59 @@ def test_vector_data_persists_across_full_stack_restart(self):
             json=batch_data,
             timeout=10,
         )
-        self.assertIn(upsert_response.status_code, [200, 201])
+        assert upsert_response.status_code in [200, 201]
 
         down_result = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)